src/model/documentselection.js

```diff
@@ -7,28 +7,20 @@
  * @module engine/model/documentselection
  */
 
-import Position from './position';
-import Range from './range';
-import LiveRange from './liverange';
-import Text from './text';
-import TextProxy from './textproxy';
-import toMap from '@ckeditor/ckeditor5-utils/src/tomap';
-import CKEditorError from '@ckeditor/ckeditor5-utils/src/ckeditorerror';
-import log from '@ckeditor/ckeditor5-utils/src/log';
-
-import Selection from './selection';
-
-const storePrefix = 'selection:';
-
-const attrOpTypes = new Set(
-	[ 'addAttribute', 'removeAttribute', 'changeAttribute', 'addRootAttribute', 'removeRootAttribute', 'changeRootAttribute' ]
-);
+import mix from '@ckeditor/ckeditor5-utils/src/mix';
+import EmitterMixin from '@ckeditor/ckeditor5-utils/src/emittermixin';
+import LiveSelection from './liveselection';
 
 /**
  * `DocumentSelection` is a special selection which is used as the
  * {@link module:engine/model/document~Document#selection document's selection}.
  * There can be only one instance of `DocumentSelection` per document.
  *
+ * `DocumentSelection` is a proxy to {@link module:engine/model/liveselection~LiveSelection} that provides
```

Comment:
Live Selection should not be public and should not be mentioned in the documentation for public classes.